### DISCUSSION TITLE
bromite add extensions

### DISCUSSION THREAD

**Discussion by ExtinctDrift19** [bromite/bromite#1609] [Ideas]
extensions installation from a chrome store

**Comment by mpaunovic** [comment]
https://www.ghacks.net/2021/11/22/how-to-get-chrome-extensions-on-android/

I haven't tried it yet but Bromite is one of the listed browsers. Now the question is does it actually work?

**Reply by ExtinctDrift19** [reply]
no

**Comment by ilpk94** [comment]
Only Kiwi browser support the estensions. And yandex, but is better kiwi

**Comment by ExtinctDrift19** [comment]
all browsers for android be able to work with extensions

**Reply by ilpk94** [reply]
But only with kiwi you can use all extensions. Yandex, no ublock origin for example. Samsung, not all extensions
